core/src/edu/asu/c3simulator/simulation/Employee.java

```diff
@@ -0,0 +1,259 @@
+package edu.asu.c3simulator.simulation;
+
+import java.util.Random;
+
+/**
+ * The Employee class is used to store all of the attributes that an employee should have.
+ * It is also used to randomly generate these attributes to apply to a list of employees
+ * available for hire.
+ * 
+ * @author nickkrogstad
+ * 
+ */
+public class Employee
+{
+	public String employeeName;
+	public static String employeePosition;
+	public int employeePay;
+	public static int employeePreferredHourlyWage;
+	public int employeeMorale;
+	public int netEarnings;
+	public float averageAnnualRaise;
+	public float averageAnnualBonus;
+	public int netBonuses;
+	public String randomEmployeePosition;
+	
+	public Employee(String name, String position, int pay, int preferredHourlyWage,
+			int morale, int net_earnings, float annualRaise, float annualBonus,
+			int net_bonus)
+	{
+		employeeName = name;
+		employeePosition = position;
+		employeePay = pay;
+		employeePreferredHourlyWage = preferredHourlyWage;
+		employeeMorale = morale;
+		netEarnings = net_earnings;
+		averageAnnualRaise = annualRaise;
+		averageAnnualBonus = annualBonus;
+		netBonuses = net_bonus;
+	}
+	
+	private Employee()
+	{
+		// TODO Auto-generated constructor stub
+	}
+	
+	/**
+	 * @return A string used display only the name and pay of an employee for the Employee
+	 *         Panel
+	 */
+	@Override
+	public String toString()
+	{
+		StringBuilder string = new StringBuilder();
+		string.append(employeeName);
+		string.append(" ($");
+		string.append(employeePay);
+		string.append(" / hr)");
+		
+		return string.toString();
+	}
+	
+	/**
+	 * @return An object of type Employee that has three randomly generated attributes:
+	 *         Name, Preferred Hourly Wage, and Position.
+	 */
+	public static Employee getRandomEmployee()
+	{
+		// TODO: Populate random employee
+		Employee employee = new Employee();
+		employee.employeeName = getRandomEmployeeName();
+		employee.employeePreferredHourlyWage = getPreferredHourlyWage();
+		employee.employeePosition = getEmployeePosition();
```

Comment:
This is difficult to follow. It looks like you're trying to access fields from a static method... 

After checking, I realize these are static methods, but it is still hard to follow, and might not do what you expect. 
Flagged for potential logic defects